examples/complete/main.tf

```diff
@@ -10,6 +10,58 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
+##############################################################################
+# Secrets Manager Certificate Setup
+##############################################################################
+
+module "secrets_manager" {
+  count                = var.existing_secrets_manager_crn == null ? 1 : 0
+  source               = "terraform-ibm-modules/secrets-manager/ibm"
+  version              = "1.18.14"
+  secrets_manager_name = "${var.prefix}-secrets-manager"
+  sm_service_plan      = "trial"
+  resource_group_id    = module.resource_group.resource_group_id
+  region               = var.region
+}
+
+module "sm_crn" {
+  source  = "terraform-ibm-modules/common-utilities/ibm//modules/crn-parser"
+  version = "1.1.0"
+  crn     = var.existing_secrets_manager_crn == null ? module.secrets_manager[0].secrets_manager_crn : var.existing_secrets_manager_crn
+}
+
+module "secrets_manager_private_cert_engine" {
+  count                     = var.existing_cert_template_name == null ? 1 : 0
+  source                    = "terraform-ibm-modules/secrets-manager-private-cert-engine/ibm"
+  version                   = "1.3.4"
+  secrets_manager_guid      = module.sm_crn.service_instance
+  region                    = var.region
+  root_ca_name              = "${var.prefix}-ca"
+  root_ca_common_name       = "${var.prefix}.${var.domain_name}"
+  intermediate_ca_name      = "${var.prefix}-int-ca"
+  certificate_template_name = "${var.prefix}-template"
```

Comment:
I suggest to make `${var.prefix}-template` a local variable, as you re-ruse it on line 56